packages/react-components/src/components/RemoteVideoTile.tsx

```diff
@@ -38,6 +46,8 @@ export const _RemoteVideoTile = React.memo(
     showLabel?: boolean;
     personaMinSize?: number;
     participantState?: ParticipantState;
+    showRemoteVideoTileContextualMenu?: boolean;
+    hostId?: string;
```

Comment:
I don't mind this optional prop here since RemoteVideoTile is an internal component. But wouldn't we have to create this prop in VideoGallery so that the RemoteVideoTiles created by the VideoGallery know their host. 

Would providing a `hostId` and seting `showRemoteVideoTileContextualMenu` to true would clicking to the ellipses or a long touch would open both the drawer menu and the contextual menu? If that's true I am still okay with that since RemoteVideoTile is internal